spec/models/work_spec.rb

```diff
@@ -197,19 +202,32 @@
         work.reload
 
         expect(fake_s3_service_pre).to have_received(:publish_files).once
-        expect(work.pre_curation_uploads).to be_empty
```

Comment:
All of these lines really just tested our mock.  No real test going on here.  Added some better tests below